Eatery/Controllers/EateryTabBarController.swift

```diff
@@ -59,4 +59,22 @@ extension EateryTabBarController: UITabBarControllerDelegate {
         return true
     }
 
+    override func tabBar(_ tabBar: UITabBar, didSelect item: UITabBarItem) {
+        let payload : Payload!
```

Comment:
Get rid of this variable and call `AppDevAnalytics.shared.log` with the payload initialized within the method call